src/algebra/category/Module/basic.lean

```diff
@@ -99,6 +125,20 @@ variables {X₁ X₂ : Type v}
 def Module.as_hom [add_comm_group X₁] [module R X₁] [add_comm_group X₂] [module R X₂] :
   (X₁ →ₗ[R] X₂) → (Module.of R X₁ ⟶ Module.of R X₂) := id
 
+notation `↟` f : 1024 := Module.as_hom f
```

Comment:
I think we better localize these notations. Notice there's a collision with the same hook notation in `category_theory/types.lean`, which should be at least noted.